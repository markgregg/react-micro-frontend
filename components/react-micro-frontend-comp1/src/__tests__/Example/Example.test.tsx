import ShallowRenderer from 'react-test-renderer/shallow';
import Examples from '../../Examples';

test("Examples snapshot", () => {
  const renderer = ShallowRenderer.createRenderer();
  renderer.render(
    <Examples
      bigChoices={[]}
      words={[]}
      typed={[]}
      complex={[]}
    />
  );
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});