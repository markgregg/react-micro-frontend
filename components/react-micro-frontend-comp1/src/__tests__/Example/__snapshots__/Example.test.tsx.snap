// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples snapshot 1`] = `
<div
  className="examples"
>
  <VerticalMenu
    onSelect={[Function]}
    options={
      [
        "Binding",
        "Single Select",
        "Multi Select",
        "Look-up",
        "Custom Values",
        "Selection prefixed",
        "Disabled",
        "Ag-Grid",
      ]
    }
    title="Catagories"
  />
  <div
    className="controls"
  />
</div>
`;

exports[`demo snapshot 1`] = `
<div
  className="examples"
>
  <VerticalMenu
    onSelect={[Function]}
    options={
      [
        "Binding",
        "Single Select",
        "Multi Select",
        "Look-up",
        "Custom Values",
        "Selection prefixed",
        "Disabled",
        "Ag-Grid",
      ]
    }
    title="Catagories"
  />
  <div
    className="controls"
  />
</div>
`;
